quantum/mcu_selection.mk

```
@@ -68,4 +67,27 @@ ifneq (,$(filter $(MCU),atmega32u4 at90usb1286))
   #     If no clock division is performed on the input clock inside the AVR (via the
   #     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
   F_USB ?= $(F_CPU)
+
+  # Interrupt driven control endpoint task
+  ifeq (,$(filter $(NO_INTERRUPT_CONTROL_ENDPOINT),yes))
+    OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+  endif
+endif
+
+ifneq (,$(filter $(MCU),atmega32a))
+  PROTOCOL = VUSB
+
+  # Processor frequency.
+  #     This will define a symbol, F_CPU, in all source code files equal to the
+  #     processor frequency in Hz. You can then use this symbol in your source code to
+  #     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+  #     automatically to create a 32-bit value in your source code.
+  F_CPU ?= 12000000
+
+  # unsupported features for now
+  NO_UART = yes
```

May be a good idea to leave these soft set? 

```suggestion
  NO_UART ?= yes
```